cmd/create/execution_util.go

```diff
@@ -10,16 +10,24 @@ import (
 	cmdGet "github.com/flyteorg/flytectl/cmd/get"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/flyteorg/flytestdlib/logger"
 	"github.com/google/uuid"
 	"sigs.k8s.io/yaml"
 )
 
 func createExecutionRequestForWorkflow(ctx context.Context, workflowName, project, domain string,
 	cmdCtx cmdCore.CommandContext) (*admin.ExecutionCreateRequest, error) {
 	// Fetch the launch plan
-	lp, err := cmdCtx.AdminFetcherExt().FetchLPVersion(ctx, workflowName, executionConfig.Version, project, domain)
-	if err != nil {
-		return nil, err
+	var lp *admin.LaunchPlan
+	var err error
+	if executionConfig.DryRun {
```

Comment:
This might stir some controversy :-D... I think `--dryRun` is ok to make server side calls as long as it doesn't have any side-effects...
e.g. [Kubectl docs](https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run)

> Dry run mode helps to evaluate a request through the typical request stages (admission chain, validation, merge conflicts) up until persisting objects to storage.

In this case, I would say retrieving something from Admin is ok to do... as long as we don't create an actual execution...

Reply:
Sure, I was thinking this would pave the way for a future dryRun=client / dryRun=server split. Now that I think a bit, it is probably a better idea to get as much coverage as possible. I added all the server-side communication that is possible without modifications.